README.md

```diff
@@ -1,31 +1,37 @@
-# JekyllPagesApi [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
+# Jekyll Pages API [![Build Status](https://travis-ci.org/18F/jekyll_pages_api.svg?branch=master)](https://travis-ci.org/18F/jekyll_pages_api)
 
-TODO: Write a gem description
+Jekyll Pages API is a [Jekyll](http://jekyllrb.com) [Plugin](http://jekyllrb.com/docs/plugins/) that generates a JSON file with data for all the Pages in your Site.
```

Comment:
The wording seems OK, but I might just put "Jekyll Plugin" all in one link to the Jekyll plugins page. You could add something like:
```
[Jekyll](http://jekyllrb.com) is a static website generator written in Ruby.
```